### PR TITLE
Fix SRCREV for all components

### PR DESCRIPTION
**Pull request by rshym**
- SRCREV is fixed for all components
- XT_KERNEL_BRANCH/XT_KERNEL_REV are removed because they should be used in the product's overrides
- SRCREV and PV are defined for the optee-test
- SRCREV selection is added for 1.11/1.15 branches of the private GSX repos

### CODE REVIEW

File: meta-xt-gateway/meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas_%.bbappend [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 RENESAS_BSP_URL = "git://github.com/xen-troops/linux.git"
-BRANCH = "${XT_KERNEL_BRANCH}"
-SRCREV = "${XT_KERNEL_REV}"
```
Comment (rshym) [inline meta-xt-gateway/meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas_%.bbappend:5]: XT_KERNEL_BRANCH/XT_KERNEL_REV should be used inside the product for DomD linux (same as it is now for the DomU linux)

File: meta-xt-gateway/meta-xt-domx-gen4/recipes-bsp/optee-test/optee-test_git.bb [hunk 2]
```diff
@@ -7,9 +7,9 @@ LIC_FILES_CHKSUM = "file://${S}/LICENSE.md;md5=daa2bcccc666345ab8940aab1315a4fa"
 BRANCH = "master"
 SRC_URI = "git://github.com/OP-TEE/optee_test.git;branch=${BRANCH};protocol=https"
 
-# SRCREV is not defined intentionaly, and should be defined in product's bbappend
+SRCREV = "21b347a3d75fd52fd49130e75c962c5b56123d2f"
```
Comment (rshym) [inline meta-xt-gateway/meta-xt-domx-gen4/recipes-bsp/optee-test/optee-test_git.bb:10]: Commit ID is taken from the meta-xt-prod-devel-rcar-gen4.

**Comment by rshym** [comment]
It's not tested yet.
The proof of the concept for the discussion.

File: meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb [hunk 3]
```diff
@@ -1,7 +1,10 @@
 
 PVRUM_URL ?= "git://git@gitpct.epam.com/epmd-aepr/pvr_um_vgpu_img.git"
 PVRUM_BRANCH = "${@bb.utils.contains('XT_USE_DDK1_11', '1', '1.11/5516664_5.1.0', '1.15/6052913_5.9.0', d)}"
-SRCREV = "${AUTOREV}"
+SRCREV = "${@bb.utils.contains('XT_USE_DDK1_11', '1',
```
Comment (lorc) [inline meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb:4]: Looking at this I realize we made a big mistake when we introduced `XT_USE_DDK1_11`. We should use the proper versioning. Like creating two different recipes , like `gles-module-egl-headers_1.11.bb` and `gles-module-egl-headers_1.15.bb`.

We need to create issue for this, I believe

**Review by lorc — COMMENTED** [review]
This PR is fine as is, but I spotted another, partially related issue.

File: meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb (hunk 3 above)
Comment (rshym) [reply meta-xt-rcar-proprietary/recipes-graphics/gles-module/gles-module-egl-headers.bb:4]: I have created it https://github.com/xen-troops/meta-xt-rcar/issues/53

**Comment by rshym** [comment]
PR is rebased on the latest master branch.

Tested by the build and the run on the h3ulcb-4x2g of the prod-devel-rcar with fixed commit on development (origin of this PR) branch.

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
`Ack` is added. Tested.
Ready to be merged.

**Pull request merged**